scripts/extract-hvd-content.ts

```diff
@@ -12,7 +12,10 @@ const env = process.env.HASHI_ENV || 'development'
 
 export const BASE_REPO_CONFIG = {
 	owner: 'hashicorp',
-	ref: process.env.CURRENT_GIT_BRANCH || 'main',
+	ref:
+		process.env.REPO !== 'hvd-docs'
+			? 'main'
+			: process.env.CURRENT_GIT_BRANCH || 'main',
```

Comment:
before the logic flow was:

if enable hvd preview on branch
then the ref is `hvd-preview`
if not
then `process.env.CURRENT_GIT_BRANCH` or `main`. 

Given that (if I understand correctly), should it be this?
```suggestion
ref:
		process.env.REPO === 'hvd-docs'
			? 'hvd-preview'
			: process.env.CURRENT_GIT_BRANCH || 'main',
```

Reply:
It was actually a bit different then that:

```
__config.flags.enable_hvd_on_preview_branch === true 
  ? 'hvd-preview'
  : process.env.CURRENT_GIT_BRANCH || 'main',
```
 
So it didn't check the repo before it checked the config flags.

Reply:
Before on dev preview, developer.hashicorp.services, it would always load hvd-docs content from the 'hvd-preview' branch. The 'hvd-preview' branch is now gone.

Reply:
ah, okay! Thanks for explaining that. I was missing that context